sonar-php-plugin/src/test/resources/org/sonar/plugins/php/phpunit/sensor/phpunit.systemout.xml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Sorry for that @vilchik-elena. Thought it would be better to use the file generated by PHPUnit with the minimum changes. So I used the same file that was throwing that error for us, replacing only system name and paths.